models/models.go

```diff
@@ -21,4 +25,10 @@ type UserResponse struct {
 	Number    string `json:"number" `
 	City      string `json:"city"`
 	Post_code string `json:"post_code" `
-}
\ No newline at end of file
+}
+
+type UserLogin struct {
```

Comment:
raczej dalbym to jako User, i umiescil wszystkie dane danego uzytkownika w strukturze.